app/src/utils/data/grantRounds.ts

```diff
@@ -231,98 +246,112 @@ export async function getGrantRoundGrantData(
   } as InitArgs);
 
   return await syncStorage(
-    grantRoundsCLRDataKeyPrefix + grantRoundAddress,
+    grantRoundsCLRDataKeyPrefix + grantRound.address,
     {
-      ts: Date.now() / 1000,
       blockNumber: blockNumber,
     },
-    async (localStorageData?: LocalStorageData | undefined, save?: () => void) => {
-      const { provider } = useWalletStore();
-      const roundGrantData = localStorageData?.data?.grantRoundCLR || {};
+    async (LocalForageData?: LocalForageData | undefined, save?: () => void) => {
+      // unpack round state
+      const roundAddress = grantRound.address;
+      const matchingTokenDecimals = grantRound.matchingToken.decimals;
+      const totalPot = BigNumber.from(grantRound.funds).toNumber();
```

Comment:
Sanity check since this is converting to a number: `grantRound.funds` is a human-readable number, e.g. 1 DAI, not a parsed number like 1e18 DAI?

Reply:
Yes, `totalPot`/`funds` should always be human-readable!

Reply:
I've added a comment for this and changed its type to `number`